control/robust.py

```diff
@@ -171,3 +171,146 @@ def hinfsyn(P,nmeas,ncon):
     CL = StateSpace(Ac, Bc, Cc, Dc)
 
     return K, CL, gam, rcond
+
+
+def _size_as_needed(w,wname,n):
+    """_size_as_needed(w,wname,n) -> w2
```

Comment:
Nonstandard docstring.  I don't think we use this type of docstring (fcn(args) -> output) anyplace else.  In addition, parameters, returns, etc are documented in a different format.  Would prefer to use a consistent style, as done in `hinfsyn` in this file.

Reply:
Will fix here and elsewhere (augw, _matrix, ...).  I see I also used this in xferfcn.py in 310f5805 in `_dc_gain_cont`.